miqi/agent/search_orchestrator.py

```
@@ -0,0 +1,155 @@
+
+        merged = "\n\n".join(search_results)
+
+        # 3. Parallel fetches (best-effort; failures never fail the search)
+        if strategy.fanout_fetches > 0 and urls:
+            targets = _dedup_urls(urls, strategy.fanout_fetches)
+            pages = await asyncio.gather(
+                *(self._safe_fetch(u) for u in targets), return_exceptions=True
+            )
+            summaries = [p for p in pages if isinstance(p, str) and p and not p.startswith('{"error"')]
+            if summaries:
+                merged += "\n\n--- 网页正文摘要（并行抓取）---\n" + "\n\n".join(summaries[: strategy.fanout_fetches])
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)